src/AdminConsole/Components/ReleaseNotes.razor

```diff
@@ -0,0 +1,7 @@
+<div class="m-4 border-2 p-3 bg-amber-50">
+    <h3>We just released v1.0.55</h3>
+    <p>You should take a look on <a href="https://github.com/passwordless/passwordless-server">Github Releases</a>!</p>
+</div>
+@code {
+    
+}
```

Comment:
Code-behind files doesn't exist, I currently don't know the 1:1 comparison to having a ctor, get and post handler.

Reply:
You can inject services at the top of the razor file, and then override a couple lifecycle methods